src/masternode/masternode-meta.cpp

```diff
@@ -56,6 +56,17 @@ CMasternodeMetaInfoPtr CMasternodeMetaMan::GetMetaInfo(const uint256& proTxHash,
     return it->second;
 }
 
+int64_t CMasternodeMetaMan::GetDsqThreshold(const uint256& proTxHash, int nMnCount)
```

Comment:
Can you add a comment explaining what this threshold is? It's not immediately obvious

Reply:
Maybe I'm being too picky, but don't comments which explain why a function exists go above the function?